src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -0,0 +1,40 @@
+package com.crud.tasks.scheduler;
+
+import com.crud.tasks.config.AdminConfig;
+import com.crud.tasks.domain.Mail;
+import com.crud.tasks.repository.TaskRepository;
+import com.crud.tasks.service.SimpleEmailService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailScheduler {
+    private static final String SUBJECT = "Tasks: Once a day email";
+
+    @Autowired
+    private SimpleEmailService simpleEmailService;
+
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @Autowired
+    private AdminConfig adminConfig;
+
+    @Scheduled(fixedDelay = 10000)
+    public void sendInformationEmail() {
+        long size = taskRepository.count();
+        simpleEmailService.send(new Mail(adminConfig.getAdminMail(), "",
+                SUBJECT, "Currently in database you got: " + printMessage(size)));
+    }
+
+    private String printMessage(long size) {
+        if (size == 0) {
```

Comment:
a co z czystym kodem? :) Dla zera jest tak samo jak dla więcej niż jednego :) haha
`...`
`private final static String MSG = "Currently in databse you have got: %d %s.";`
`...`
`simpleemailService.send(..., String.format(MSG, size, task(size));`


`private String task(long size) {`
`return size == 1 ? "task" : "tasks";`
`}`